### DISCUSSION TITLE
Set tags when executing pipelines via GraphQL API

### DISCUSSION THREAD

**Discussion by andreashofm** [dagster-io/dagster#4032] [Q&A]
Is it possible to set tags when launching a pipeline run via the GraphQL API?

The [examples](https://docs.dagster.io/concepts/dagit/graphql#launch-a-pipeline-run) indicates that the `ExecutionParams` do not include the `tags`. Is this true? And if so, is it possible to include them easily?

**Comment by catherinewu** [comment]
Hi @andreashofm! Yes, you should be able to set tags via the `tags` field of `executionMetadata` in `ExecutionParams`

Code link: https://github.com/dagster-io/dagster/blob/master/js_modules/dagit/packages/core/src/graphql/schema.graphql#L644

**Reply by andreashofm** [reply]
Thank you very much, @catherinewu ! That is exactly what I was searching for.
